webapi/src/main/scala/org/knora/webapi/core/AppServer.scala

```diff
@@ -104,6 +104,7 @@ final case class AppServer(
                  ZIO.logError("IIIF service not running") *> ZIO.die(new Exception("IIIF service not running"))
              }
              .when(requiresIIIFService)
+             .orDie
```

Comment:
this is of course a discussion for another PR... but should we really die without retry first?

Reply:
IMHO dying during the startup is acceptable for now.